Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mambaorg/micromamba
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+RUN micromamba install -y -c https://packages.qiime2.org/qiime2/2023.2/tested/ \
+	-c conda-forge -c bioconda -c defaults \
+	q2cli q2-fondue
+ENV PATH /opt/conda/bin:$PATH
+
+RUN echo $(whoami)
+RUN uname=$(whoami)
+
+USER root
+RUN apt-get update && apt-get install uuid-runtime
+RUN mkdir $HOME/.ncbi
+RUN printf '/LIBS/GUID = "%s"\n' `uuidgen` > $HOME/.ncbi/usert-settings.mkfg
```

Comment:
Hey @lina-kim, I think the issue described by @adamovanja is due to the fact that you first run these two commands _but_ then you change the user (line 16), effectively ignoring the changes done by those commands. I think it should be possible to move those two between lines 16 and 17 - they shouldn't need root privileges.

Reply:
Oh interesting! Thanks for your input. Hmm I had errors in this step running as a standard user, but I will look into this again.

Reply:
Sadly moving these lines after the user change did not resolve the issue. Thanks for the suggestion! I will incorporate a different workaround.